tests/CacheCompat/CommonCache.Test.MsalNode/NodeScript/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "msal-node-username-password",
+    "version": "1.0.0",
+    "description": "Command line app that uses Oauth password grant flow to get a token from Azure AD",
+    "main": "index.js",
+    "private": true,
+    "scripts": {
+        "start": "node index.js --inputPath c:\\temp\\input.json",
+        "build:package": "cd ../../../lib/msal-common && npm run build && cd ../msal-node && npm run build",
+        "start:build": "npm run build:package && npm start"
+    },
+    "author": "Microsoft",
+    "license": "MIT",
+    "dependencies": {
+        "@azure/msal-node": "latest"
```

Comment:
@sameerag - I think this also catches pre-release versions, pls correct me if I'm wrong